src/main/resources/liquibase/db.changelog-1.3.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+
+    <changeSet id="1.3.0 add new column" author="serhii.ponomarenko" >
```

Comment:
It would be great not to create a new changelog for such minor changes. Could you update it and move it to the bottom of 1.2 changelog?